vscode/src/azure/commands.ts

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as vscode from "vscode";
+import { log } from "qsharp-lang";
+
+import {
+  Job,
+  Target,
+  WorkspaceTreeItem,
+  WorkspaceTreeProvider,
+} from "./treeView";
+import {
+  getJobFiles,
+  getTokenForWorkspace,
+  queryWorkspace,
+  queryWorkspaces,
+  submitJob,
+} from "./workspaceActions";
+import { QuantumUris, compileToBitcode } from "./networkRequests";
+import { getQirForActiveWindow } from "../qirGeneration";
+
+// TODO: Proper docs on the qsharp wiki
+const corsDocsUri =
+  "https://gist.github.com/billti/09637269db4bae86c0e3a552dd20eb9b";
```

Comment:
Shall I make an `aka.ms` real quick so we can redirect this after we ship if need be?

Reply:
Updated to point to the wiki. We can update with a aka.ms redirect later if needed (hopefully the service will fix the need for CORS changes well before we ship anyway)